applications/sasquatch/charts/app-metrics/values.yaml

```diff
@@ -13,7 +13,14 @@ globalAppConfig:
       - "username"
   mobu:
     influxTags:
-      - "type"
+      - "sync"
+      - "success"
+      - "flock"
+      - "business"
+      - "username"
+      - "notebook"
+      - "repo"
+      - "repo_ref"
```

Comment:
I would leave this off, particularly if we go with adding the hash of the repo, but even if not. I don't think we're going to routinely run checks from multiple refs from the same repo in the same environment in circumstances where we care about metrics. I'm betting the only values will be whatever mobu runs by default and then a totally random variety of development branches from the GitHub verification.

Reply:
I ditched `repo_ref`, and added `repo_hash`, but [you convinced me that it doesn't need to be a tag](https://github.com/lsst-sqre/mobu/pull/387#discussion_r1887672199).